distribution/docker/src/docker/Dockerfile

```diff
@@ -284,9 +232,7 @@ LABEL org.label-schema.build-date="${build_date}" \\
   org.opencontainers.image.url="https://www.elastic.co/products/elasticsearch" \\
   org.opencontainers.image.vendor="Elastic" \\
   org.opencontainers.image.version="${version}"
-<% } %>
 
-<% if (docker_base == 'ubi') { %>
 LABEL name="Elasticsearch" \\
   maintainer="infra@elastic.co" \\
   vendor="Elastic" \\
```

Comment:
@breskeby I've tried to keep the logic unchanged, but here I have decided to change it to simplify the logic a bit.
Is it OK that wolfi Docker image will also have these labels?